cli/dcoscli/tables.py

```diff
@@ -112,7 +115,10 @@ def get_deployment(app):
         ("CMD", get_cmd)
```

Comment:
@kensipe and I discussed about introducing a general helper to enable this truncating feature. If this is too much, I just change to use the truncate functionality only for `cmd`. What do you think @tamarrow ?